<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多路监控客户端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="rootLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="selectPage">
       <layout class="QVBoxLayout" name="selectLayout">
        <item>
         <spacer name="topSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="selectBtnLayout">
          <item>
           <spacer name="leftSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectMain">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>120</height>
             </size>
            </property>
            <property name="text">
             <string>主码流
H.265  1920×1080</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="midSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectSub">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>120</height>
             </size>
            </property>
            <property name="text">
             <string>子码流
H.264  640×360</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="rightSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="bottomSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="settingsLayout">
          <item>
           <spacer name="settingsLeftSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSettings">
            <property name="text">
             <string>设置截图目录</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="settingsRightSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="videoPage">
       <layout class="QVBoxLayout" name="videoPageLayout">
        <item>
         <widget class="QLabel" name="videoLabel">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="text">
           <string>未连接</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="controlLayout">
          <item>
           <widget class="QPushButton" name="btnBack">
            <property name="text">
             <string>返回</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConnect">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDisconnect">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>断开</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnScreenshot">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>截图</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRecord">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>录像</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
